code/modules/admin/admin_verbs.dm

```diff
@@ -1159,7 +1249,8 @@
 		if(isobserver(C.mob) || C.mob.stat == DEAD)
 			continue
 		if(isxeno(C.mob))
-			C.mob.loc = get_turf(usr)
+			var/mob/living/carbon/Xenomorph/X = C.mob
```

Comment:
Same here, substitute `isobserver(C.mob)` with `!isxeno(C.mob)`, remove the type cast and avoid unnecessary checks.
If the compiler complains about the proc not being defined for the variable just type cast it at the beginning and use a `!istype()`early continue or something.